src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/custom.py

```diff
@@ -32,41 +47,68 @@ def cli_consumption_list_usage(cmd, client, billing_period_name=None, top=None,
         filter_to = "properties/usageEnd le \'{}\'".format(end_date.strftime("%Y-%m-%dT%H:%M:%SZ"))
         filter_expression = "{} and {}".format(filter_from, filter_to)
     if top:
-        pages = client.list(scope, expand=expand, filter=filter_expression, top=top)
+        pages = client.list_by_billing_period(billing_period_name, expand=expand, filter=filter_expression, top=top)
         return list(pages.advance_page())
-    return list(client.list(scope, expand=expand, filter=filter_expression))
+    return list(client.list_by_billing_period(billing_period_name, expand=expand, filter=filter_expression))
 
 
-def cli_consumption_list_reservations_summaries(client, grain, reservationorderid, reservationid=None, start_date=None, end_date=None):
-    """List all the reservation summaries """
-    if reservationid:
-        scope = "providers/Microsoft.Capacity/reservationorders/{}/reservations/{}".format(reservationorderid, reservationid)
-    else:
-        scope = "providers/Microsoft.Capacity/reservationorders/{}".format(reservationorderid)
+def cli_consumption_list_reservations_summaries(client, grain, reservationorderid, start_date=None, end_date=None):
```

Comment:
You should change 'reservationorderid' to 'reservation_order_id'. It's more readable, and will also adding the parameter option '--reservation-order-id' by default.